xdem/examples.py

```diff
@@ -113,7 +100,7 @@ def mkdir_p(out_dir):
                                 transform=reference_raster.transform, crs=reference_raster.crs)
 
     # Save it so that future calls won't need to recreate the file
-    mkdir_p(os.path.dirname(FILEPATHS_PROCESSED['longyearbyen_ddem']))
+    pathlib.Path(os.path.dirname(FILEPATHS_PROCESSED['longyearbyen_ddem'])).mkdir(parents=True, exist_ok=True)
```

Comment:
Wouldn't just `os.makedirs(..., exist_ok=True)` do the trick? Not that it matters!

Reply:
You are right, even better

Reply:
I still have those bad habits from coding in old Python